test/e2e/metamask-ui.spec.js

```diff
@@ -279,6 +279,7 @@ describe('MetaMask', function () {
       await gasPriceInput.fill('20');
       await driver.delay(veryLargeDelayMs);
       await driver.clickElement({ text: 'Save', tag: 'button' });
+      await driver.delay(veryLargeDelayMs);
```

Comment:
You'll see some of these pattern in e2e tests in this PR where I have to call delay after Save is clicked. This is because it takes some milliseconds between when Save is clicked and when the values are updated and if the next line is executed immediately, it will use the old values.

Reply:
In future cases, where possible we can use the following method to wait for an expected condition (e.g. updated gas estimate), instead of adding a delay

```
await driver.waitForSelector({
	css: 'GAS_ESTIMATE_CSS_SELECTOR',
	text: 'GAS_ESTIMATE_UPDATED_VALUE',
});
```